frontend/src/metabase/visualizations/components/TableSimple/TableFooter.tsx

```diff
@@ -84,6 +85,7 @@ const TableFooter = forwardRef<HTMLDivElement, TableFooterProps>(
       >
         <PaginationMessage>{paginateMessage}</PaginationMessage>
         <PaginationButton
+          className={CS.textPrimary}
```

Comment:
The table pagination was never colored. It's been using the default color (black) forever.

![Screenshot 2024-07-05 at 11 05 59 PM](https://github.com/metabase/metabase/assets/1937582/0983b2b1-f1dc-4082-9ea1-103b712fe40b)

#### Before
![image](https://github.com/metabase/metabase/assets/1937582/54851daa-1788-4d0f-b155-fa66f7210516)
![image](https://github.com/metabase/metabase/assets/1937582/67a83a96-a65a-4610-ad4d-31d52a7a7e44)

#### After
![image](https://github.com/metabase/metabase/assets/1937582/17a81084-f104-4c1c-a22c-d01957c3acd3)
![image](https://github.com/metabase/metabase/assets/1937582/784d92de-edac-4958-a8f1-9fdff4b5f430)